editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2957,6 +2957,11 @@ void CanvasItemEditor::_draw_ruler_tool() {
 		Point2 corner = Point2(begin.x, end.y);
 		Vector2 length_vector = (begin - end).abs() / zoom;
 
+		const real_t horizontal_angle_rad = length_vector.angle();
+		const real_t vertical_angle_rad = Math_PI / 2.0 - horizontal_angle_rad;
+		const int horizontal_angle = round(180 * horizontal_angle_rad / Math_PI);
+		const int vertical_angle = round(180 * vertical_angle_rad / Math_PI);
```

Comment:
Why was this moved out of `draw_secondary_lines` condition? None of these variables are used outside.

Reply:
There are two  `draw_secondary_lines`  if statements.  The first one to draw the lines, and the second one for the text. But `horizontal_angle_rad` and `vertical_angle_rad` are used to calculate `horizontal_angle` and `vertical_angle`, so those need to stay in the wider scope I think. But I can move `horizontal_angle` and `vertical_angle` under the second if statement.